lib/ansible/modules/cloud/amazon/ecs_tag.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Michael Pechner <mikey@mikey.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: ecs_tag
+short_description: create and remove tags on ecs clusters.
+notes:
+    - none
+description:
+    - Creates, removes and lists tags for an ecs_cluster.
+       The resource is referenced by its cluster name.  It is designed to be used with complex args (tags), see the examples.
+version_added: '2.10'
+author:
+  - Lester Wade (@lwade)
+  - Paul Arthur (@flowerysong)
+  - Michael Pechner (@mpechner)
+requirements: [ boto3, botocore ]
+options:
+  cluster_name:
+    description:
+      - The name of the cluster who's resources we are tagging
+    required: true
+    type: str
+  resource:
+    description:
+      - The ecs resource name. Can be none only if resource_type is cluster.
+    required: false
+    type: str
+  resource_type:
+    description:
+      - The type of resource
+    default: cluster
+    choices: ['cluster', 'task', 'service', 'task_definition', 'container']
+    type: str
+  state:
+    description:
+      - Whether the tags should be present or absent on the resource. Use list to interrogate the tags of a ecs resource.
+    default: present
+    choices: ['present', 'absent', 'list']
+    type: str
+  tags:
+    description:
+      - A dictionary of tags to add or remove from the resource.
+      - If the value provided for a tag is null and C(state) is I(absent), the tag will be removed regardless of its current value.
```

Comment:
```suggestion
      - If the value provided for a tag is null and I(state=absent), the tag will be removed regardless of its current value.
```

C is for values I is for parameters (and can be used for param=) there are some bad examples around.